### ISSUE TITLE
Feature Request: path to repository

### ISSUE THREAD

**Issue by ruzz311** [mateodelnorte/meta#47]
**TL;DR** Add property to ```.meta``` configuration to specify a local path to each project.

___

I just found this project and I'm really intrigued with integrating it into my development workflow.  The problem I find is that I have a ton of existing repos with various stashes and remotes added to each.  I could just make a meta project in the root of all my sub-projects but if I wanted to have multiple meta configs for different project groupings I would not be able to.  And while I think there's merit to doing a little fall cleaning on all these, I'd love to be able to use meta without a huge change to environment vars and scripts I have for various command line tools.

Additionally, there are certain languages _&#42;cough_ _go-lang_ _cough&#42;_ that have a specific directory that they need to live in.  Sure there are ways around it but there can be weird side-effects to this.

As a user of meta I'd like to be able to edit the .meta JSON to include a local path to each git repo so I may store them where I please.

**Comment by mateodelnorte** [comment]
You can already do this. The key for each repo doesn't have to just be a
folder in the direct directory you're in. It can be a relative path to
anywhere.
